Code/Editor/WelcomeScreen/WelcomeScreenDialog.cpp

```diff
@@ -79,12 +80,18 @@ WelcomeScreenDialog::WelcomeScreenDialog(QWidget* pParent)
     {
         projectPreviewPath = ":/WelcomeScreenDialog/DefaultProjectImage.png";
     }
+
+    qreal screenDpiFactor = QHighDpiScaling::factor(screen());
+
+    QSize imageSize = ui->activeProjectIcon->size();
+    imageSize.setWidth(aznumeric_cast<int>(aznumeric_cast<qreal>(imageSize.width()) * screenDpiFactor));
+    imageSize.setHeight(aznumeric_cast<int>(aznumeric_cast<qreal>(imageSize.height()) * screenDpiFactor));
+
+    QPixmap projectImage = QPixmap(projectPreviewPath);
+    projectImage.setDevicePixelRatio(screenDpiFactor);
+    
     ui->activeProjectIcon->setPixmap(
-        QPixmap(projectPreviewPath).scaled(
-            ui->activeProjectIcon->size(),
-            Qt::KeepAspectRatioByExpanding,
-            Qt::SmoothTransformation
-        )
+        projectImage.scaled(imageSize, Qt::KeepAspectRatioByExpanding, Qt::SmoothTransformation)
```

Comment:
Any reason why the aspect ratio flag here is different? I figured the ignore would always be safe since we're scaling both dimensions by a uniform factor.

Reply:
AFAIK, the different flag in the splashscreen and about other other widgets were due to the fact that the scaling used to squish the image a bit since the images are used as background of a widget that can ultimately be resized. That flag ensured the image was rendered ok in most cases.
I can likely just get that as an argument for the helper so every widget can still use the flag they prefer.